src/utils/mdast-to-vdom.js

```diff
@@ -220,7 +220,7 @@ class VDOMTransformer {
       if (child.type === 'raw') {
         if (child.value.firstElementChild === null) {
           if (stack.length === 0) {
-            throw new Error(`no matching inline element found for ${child.html}`);
+            // ignore unmatched inline elements
```

Comment:
Would it make sense to somehow log a info/warning to track these, so content authors can ultimately fix the issue?

Reply:
yes, however, I don't think that authors will ever see those messages.